frontend/src/components/AppointmentSchedule.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import {Calendar, momentLocalizer} from 'react-big-calendar'
+import moment from 'moment'
+import 'react-big-calendar/lib/css/react-big-calendar.css'
+
+const localizer = momentLocalizer(moment)
+
+export default () => {
+
+  return (
```

Comment:
Could you please stick to 2-character indentation? I know it may sound petty, but nothing ruins `git blame` as easily as code that would get reformatted all over the place at the first possible opportunity.